### DISCUSSION TITLE
Welcome to Awesome-Automations Discussions!

### DISCUSSION THREAD

**Discussion by Arsenic-ATG** [Arsenic-ATG/Awesome-Automations#56] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Help others.
  * Share new ideas ( regarding automations or repository itself )
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by UG-SEP** [comment]
Hi @Arsenic-ATG,
How can we connect with each other do you use LinkedIn
As well I have sent you a friend request in discord please go and check it Discord Name: 
> UG-SEP

**Reply by Arsenic-ATG** [reply]
yes, I am on linkedln also but I am very sparsely active on it ( I think it has been over an year now, since I lasted opened linkedln )

as for discord, I didn't receive any friend request ( or maybe didn't noticed it was you and rejected it while cleaning up my friend requests ), can you send it again ?

**Reply by UG-SEP** [reply]
ok

**Reply by UG-SEP** [reply]
I send it again @Arsenic-ATG 
please check it
